Add Beats Per Minute stub (#3232)
ppy/osu-wiki: wiki/Beatmapping/Beats_per_minute/en.md

---
tags:
  - BPM

>   - Tempo

Reviewer: ```suggestion
  - tempo
```
doesnt matter lol but might as well be consistent since everything else is lowercase already.

---

# Beats per minute

> **Beats Per Minute**, abbreviated as *BPM*, refers to the number of whole beats in a minute. BPM is used to measure the [tempo](wikipedia.org/wiki/Tempo) of a song. This is needed to line up the beats of a song with the [hit objects](/wiki/Hit_Objects) of a [beatmap] successfully. It can be changed in the [beatmap editor](/wiki/Beatmap_Editor) under the `Timing` tab, or by using the shortcut (`Ctrl` + `P`).

Reviewer 2: ```suggestion
**Beats per minute**, abbreviated as *BPM*, refers to the number of whole beats in a minute. This is used to measure the [tempo](https://en.wikipedia.org/wiki/Tempo) of a song, which is then used to place [hit objects](/wiki/Hit_Objects) in a [beatmap](/wiki/Beatmap). Setting the BPM is the second thing that needs to be done when creating a new beatmap—the first being [song setup](/wiki/Beatmap_Editor/Song_Setup). It can be changed later in the [beatmap editor](/wiki/Beatmap_Editor) under the `Timing` tab, or by pressing `Ctrl` + `P`.
```

> Most modern songs use a single BPM which requires a single timing point. More complex songs may require multiple timing points and timing resets to adjust succesfully. Timing points are shown as red lines in the timeline at the bottom of the editor.

Reviewer 2: ```suggestion
Most modern songs use a single BPM which requires a single timing point. More complex songs may require multiple timing points and timing resets to adjust successfully. Timing points are shown as red lines in the timeline at the bottom of the editor.
```
suggestion fixes a typo, but can you elaborate or explain what you mean by `timing resets to adjust successfully`?

Author: should be "to be timed correctly". timing resets and timing points should each have their own articles tbh.
